### PR TITLE
Update pika consumer code snippet

### PR DESCRIPTION
**Pull request by rschuetzler**
**Proposed changes**:
- Update code snippet for the Pika consumer in Event Brokers docs

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @joejuzl will take a look at it as soon as possible ✨

**Review by joejuzl — CHANGES_REQUESTED** [review]
Thanks for the PR.
Could you please give a quick explanation of the changes and why they are needed?

**Comment by rschuetzler** [comment]
The `pika` API has changed slightly, and the changes bring the code snippet up to date with the latest version. Both the structure of the `basic_consume()` API call and the callback method signature have changed.

File: docs/docs/event-brokers.mdx [hunk 1]
```diff
@@ -84,26 +84,25 @@ import json
...
-        pika.ConnectionParameters('rabbit', credentials=credentials))
+        pika.ConnectionParameters("rabbit", credentials=credentials)
+    )
 
     # start consumption of channel
     channel = connection.channel()
-    channel.basic_consume(_callback,
-                          queue='rasa_events',
-                          no_ack=True)
+    channel.basic_consume(queue="rasa_events", on_message_callback=_callback)
```
Comment (joejuzl) [inline docs/docs/event-brokers.mdx:105]: What about `no_ack=True`? Seems like it's been renamed to `auto_ack` - do we want to set that to `True` too?

**Review by joejuzl — CHANGES_REQUESTED** [review]

File: docs/docs/event-brokers.mdx (hunk 1 above)
Comment (rschuetzler) [reply docs/docs/event-brokers.mdx]: Yes indeed we do. Sorry about that. I was thinking that `auto_ack=True` would be the opposite of `no_ack=True`, when in fact they are the same. Updated.

**Review by joejuzl — APPROVED** [review]
Great - thanks for this!

**Comment by rschuetzler** [comment]
Is there anything I need to do to get this merged?

**Pull request merged**